client/src/resources/lang/en_us.json

```diff
@@ -513,7 +513,7 @@
     "wiki.article.standard.visit.title": "Visit",
     "wiki.article.standard.visit.text": "When you select someone, you will usually visit them. Visits are what actually causes active abilities to work. For example: A Detective could select a player, but a Witch could cause them to visit someone else, and then the Detective will unknowingly get their results from the player that they visited, but not the player they selected.\n\nVisits can have properties such as astral or attacking.\n   - An astral visit is undetected by most roles.\n    - An attacking visit will usually deal an attack to the player being visited, bodyguards and trappers abilities will work differently if the visit is an attack.",
     "wiki.article.standard.convert.title": "Convert",
-    "wiki.article.standard.convert.text": "A player's role changes during the game. All of their ability uses are refilled to maximum. _For Example:_\n- If someone is converted to a Veteran, they get all three alerts.\n- A Vampire can convert players into vampires.",
+    "wiki.article.standard.convert.text": "A player's role changes during the game. All of their ability uses are refilled to maximum. _For Example:_\n- If someone is converted to a Veteran, they get all three alerts.\n- A Prophet can convert players into cultists.",
```

Comment:
Can there be more than one? Should be "The Prophet" if not

Reply:
Fixed